pkg/services/updates.go

```diff
@@ -223,7 +223,7 @@ func (s *UpdateService) RebootDevice(update *models.UpdateTransaction) {
 		filePath := cfg.TemplatesPath
 		templateName := "template_playbook_dispatcher_reboot_device.yml"
 		template, err := template.ParseFiles(filePath + templateName)
-		fmt.Printf("template: %v", template)
+		log.Infof("template: %v", template)
```

Comment:
```suggestion
		log.Debugf("template: %v", template)
```

Also debug here? Looks useful for development but not so much for prod.